module/VuFind/src/VuFind/Db/Table/LoginToken.php

```diff
@@ -66,41 +66,6 @@ public function __construct(
         parent::__construct($adapter, $tm, $cfg, $rowObj, $table);
     }
 
-    /**
-     * Save a token
-     *
-     * @param int    $userId    User identifier
-     * @param string $token     Login token
-     * @param string $series    Series the token belongs to
-     * @param string $browser   User browser
-     * @param string $platform  User platform
-     * @param int    $expires   Token expiration timestamp
-     * @param string $sessionId Session associated with the token
-     *
-     * @return LoginTokenRow
-     */
-    public function saveToken(
```

Comment:
Since this method has never been included in released code, I decided it was safe to delete it now that the logic has been refactored to the LoginTokenService, rather than going to the trouble of retaining a deprecated version.